spec/PeriodicBackgroundSync-index.bs

```diff
@@ -204,3 +205,9 @@ The <code><dfn method for=PeriodicSyncManager title="register(tag, options)">reg
         1. Set |currentRegistration|'s associated [=state=] to [=pending=].
     1. Else, if |serviceWorkerRegistration|'s [=registration state=] is [=firing=], set |serviceWorkerRegistration|'s [=registration state=] to [=reregisteredWhileFiring=].
     1. [=Resolve=] |promise|.
+
+The <code><dfn method for=PeriodicSyncManager title="getTags()">getTags()</dfn></code> method when invoked, MUST return [=a new promise=] |promise| and run the following steps [=in parallel=]:
+1. Let |serviceWorkerRegistration| be the {{PeriodicSyncManager}}'s associated [=service worker registration=].
+1. Let |currentTags| be a new [=list=].
+1. For each |registration| in |serviceWorkerRegistration|'s [=list of periodicsync registrations=], add |registration|'s [=tag=] to |currentTags|.
```

Comment:
|registration| of |serviceWorkerRegistration|'s ...

`in` does a different thing

Reply:
what does in do? I thought it's plain English.
Example text here (see number 7):
https://wicg.github.io/BackgroundSync/spec/#sync-manager-interface

Reply:
Infra spec example uses `of`: https://infra.spec.whatwg.org/#list-iterate

It also explains how these concepts are closely related to the JavaScript language. In JS, `of` is used to iterate over lists, whereas `in` is used to iterate over an object's properties.

Reply:
done.